.circleci/config.yml

```diff
@@ -14,6 +14,7 @@ jobs:
     steps:
       - checkout
       - setup_remote_docker:
+          version: 17.11.0-ce
```

Comment:
This is the latest version they support? 😞

Reply:
from the doc looks like is the most recent, is under "edge" :D
Did not try to configure above the doc, maybe is just outdated..

Reply:
I'm sure @dnephin knows 😅

Reply:
tried actually the latest 18.05 and seems to be scheduled the job